webpack.dev.js

```diff
@@ -5,7 +5,7 @@ const webpackBase = require('./webpack.config')
 module.exports = Object.assign(webpackBase, {
   devtool: 'inline-source-map',
   devServer: {
-    port: 3000,
+    port: 3001,
```

Comment:
Is this a local dev change that slipped into the PR on accident?

Reply:
Yeah, you are right, reverted in [`b8b0bcc` (#72)](https://github.com/smartcontractkit/operator-ui/pull/72/commits/b8b0bccab5bf4387574810e505452aefd8d885f0)